core/modules/wdb/QueryRunner.cc

```diff
@@ -126,6 +126,18 @@ void QueryRunner::_setDb() {
     }
 }
 
+
+std::mutex totalMemWaitMtx;
+double totalMemWaitTime(0);
+double memWaitCallCount(0);
+std::atomic<uint64_t> lessThan1(0);
+std::atomic<uint64_t> lessThan5(0);
+std::atomic<uint64_t> lessThan10(0);
+std::atomic<uint64_t> lessThan20(0);
+std::atomic<uint64_t> lessThan40(0);
+std::atomic<uint64_t> over40(0);
```

Comment:
This set of variables could be turned into a vector of atomics as explained in:
https://stackoverflow.com/questions/13193484/how-to-declare-a-vector-of-atomic-in-c?lq=1

This may add more flexibility to the code and make it significantly smaller (provided that wrapper class around the atomic is put into a separate header sitting in module utility::).